hack/e2e-scripts/qa-container-scale.sh

```diff
@@ -84,7 +85,22 @@ while (( "$#" )); do
       NEXD_COUNT="$2"
       shift 2
       ;;
-    -help)
+    --deployment|-d)
+      case "$2" in
+        "prod")
+           NEXODUS_API_SERVER="https://try.nexodus.io"
+           NEXODUS_AUTH_SERVER="https://auth.try.nexodus.io"
+           ;;
+        "playground")
+           NEXODUS_API_SERVER="https://playground.nexodus.io"
+           NEXODUS_AUTH_SERVER="https://auth.playground.nexodus.io"
+           ;;
+        *)
+           ;;
```

Comment:
I'd prefer this checked for `qa`, and then under `*` it printed the help text and exited non-zero.